myScript.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS Notes(
```

Comment:
Please restructure table order and test cases with referencing non-existing table